mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -700,15 +544,28 @@ std::vector<InitParamsAccel>
 PopulateParamsXDL::getTuningParameters(KernelType opType, Type dataTypeA,
                                        Type dataTypeB, StringRef arch) const {
   ArrayRef<InitParamsAccel> params;
-  switch (dataTypeA.getIntOrFloatBitWidth()) {
-  case 8:
-    params = {initParametersForward8Bit, nInitParametersForward8Bit};
-    break;
-  case 16:
-    params = {initParametersFp16, nInitParametersFp16};
-    break;
-  default:
-    params = {initParameters, nInitParameters};
+  if (opType == KernelType::Gemm) {
+    switch (dataTypeA.getIntOrFloatBitWidth()) {
+    case 8:
```

Comment:
this would apply to both f8 and i8 here, is that the goal?

Reply:
Here I 've just add select conv or gemm list based on opType. I think we're using non-accel configs for fp8